packages/api-server/api_server/repositories/tasks.py

```diff
@@ -162,32 +164,39 @@ async def query_task_requests(self, task_ids: List[str]) -> List[DbTaskRequest]:
             raise HTTPException(422, str(e)) from e
 
     async def save_task_state(self, task_state: TaskState) -> None:
-        task_state_dict = {
-            "data": task_state.json(),
-            "category": task_state.category.__root__ if task_state.category else None,
-            "assigned_to": task_state.assigned_to.name
-            if task_state.assigned_to
-            else None,
-            "unix_millis_start_time": task_state.unix_millis_start_time
-            and datetime.fromtimestamp(task_state.unix_millis_start_time / 1000),
-            "unix_millis_finish_time": task_state.unix_millis_finish_time
-            and datetime.fromtimestamp(task_state.unix_millis_finish_time / 1000),
-            "status": task_state.status if task_state.status else None,
-            "unix_millis_request_time": task_state.booking.unix_millis_request_time
-            and datetime.fromtimestamp(
-                task_state.booking.unix_millis_request_time / 1000
-            ),
-            "requester": task_state.booking.requester
-            if task_state.booking.requester
-            else None,
-        }
+        # FIXME: this may be fixed upstream in DB or ORM, this mutex can be
+        # removed once these libraries have been updated and tested to be fixed
```

Comment:
Can you add/document a manual test (due to it's nature, we probably don't want to run the test in ci) that can reproduce this?

Reply:
https://github.com/open-rmf/rmf-web/pull/928/commits/cf37addbef64dcd00d6fd7bf0de7994682e4c651